index.js

```diff
@@ -99,14 +99,15 @@ MenuDroppoComponent.prototype.manageListeners = function() {
 
 MenuDroppoComponent.prototype.componentDidMount = function() {
   if (this) {
-    window.addEventListener('click', this.windowWasClicked.bind(this))
+    this.windowClickHandler = this.windowWasClicked.bind(this);
+    window.addEventListener('click', this.windowClickHandler)
     var container = findDOMNode(this)
     this.container = container
   }
 }
 
 MenuDroppoComponent.prototype.componentWillUnmount = function() {
-  window.removeEventListener('click', this.windowWasClicked.bind(this))
+  window.removeEventListener('click', this.windowClickHandler)
```

Comment:
We fix this in L102 by assigning the bound function to a separate variable. When re-using it, `window.removeEventListener` can perform the correct equality check and remove the listener.

This follows existing patterns in the lib: see [outsideClickHanlder](https://github.com/flyswatter/menu-droppo/pull/1/files#diff-168726dbe96b3ce427e7fedce31bb0bcR94).